src/app/wizards/dataClass/data-class-step2/data-class-step2.component.ts

```diff
@@ -190,7 +191,14 @@ export class DataClassStep2Component
 
       invalid = this.myForm.invalid;
     }
-    if (this.model.createType === 'copy') {
+    if (['copy', 'import', 'extend'].includes(this.model.createType)) {
```

Comment:
I think `createType` could benefit from using a union string type here to ensure the string is always correct, e.g.:

```ts
export type CreateType = 'copy' | 'import' | 'extend';
```